improve autokit quickstart (#1017)
balena-os/leviathan: documentation/quickstart-autokit.md

# Getting Started with Autokit Worker 

This is a quick start guide for using the leviathan framework with the autokit worker.

## As a prerequisite

1. Make sure you have built your autokit using the following [guide](https://github.com/balena-io-hardware/autokit-assembly-doc)
2. Go through the steps listed on the {@page Quickstart | Quickstart Page}

## Provision your autokit into a leviathan-worker fleet

To use the autokit with the leviathan framework, your autokit must be running a [leviathan-worker](https://github.com/balena-os/leviathan-worker) ctontainer. To do this, we recommed provisioning the autokit host to a balena fleet:

1. If you don't already have access to a fleet, [create one](https://docs.balena.io/learn/getting-started/raspberrypi3/nodejs/#create-a-fleet)
2. Navigate to the [leviathan-worker](https://github.com/balena-os/leviathan-worker) repository. Clone it, and from that repository use `balena push` to [push the release to your fleet](https://docs.balena.io/learn/deploy/deployment/#overview)

> 3. [Provision](https://docs.balena.io/learn/deploy/deployment/#overview) your autokit to this fleet. Depending on the device-type of your autokit host, the process may differ. Ensure you follow the correct provisioning procedure for your device type. *Ensure that you provision the device as using ethernet only*

Reviewer: ```suggestion
3. [Provision](https://docs.balena.io/learn/deploy/deployment/#overview) your autokit to this fleet. Depending on the device-type of your autokit host, the process may differ. Ensure that you provision the device using the image configured to use **ethernet only**. 
```

> 4. Ensure the public URL is enabled on your device.

Reviewer: ```suggestion
4. When the device is successfully provisioned on balenaCloud, make sure to enable the public URL for your device.
```

5. Add the appropriate environment variables to the device. You may have to adjust these depending on the autokit setup and the device under test. The main ones, relevant to the leviathan-worker are:
   1. `TESTBOT_DUT_TYPE` must be set to the device type slug of the DUT. See supported devices [here] - although if your device is not there, you can pick another DUT with the closest procedure and use that instead. For example for a simple sd card boot DUT, use `raspberrypi4-64`, and for a flasher image use `intel-nuc`.
   2. `WORKER_TYPE` must be set to `autokit`

>    3. The appropriate [autokit setup variables](https://github.com/balena-os/leviathan-worker/blob/master/lib/workers/autokit.ts#L17), if any of the hardware differs from the default. Cross reference these strings with the [autokit feature implementations](https://github.com/balena-io-hardware/autokit-interface-sw/tree/master/lib/features).

Reviewer: How about we convert this into a markdown table?

   4. Any other relevent autokit variables must be set, if the default values aren't appropriate
      1. `WIFI_IF` - the name of the primary wifi interface of the autokit host (default `wlan0`)
      2. `WIRED_IF` - the name of the USB-ethernet interface of the autokit host - this can be found by going into the autokit host over ssh and finding it (default `eth0`)

>       3. `DEV_SERIAL` - the name of the USB-serial interface of the autokit (default `/dev/ttyUSB0`)

Reviewer: same suggestion as above

>    5. Ensure that the `worker` container on your autokit host has started. In the dashboard logs window, you should see the message `worker setup completed`. If the container is not starting, the most likely cause is either hardware is not plugged in, or env vars are not set up correctly.

Reviewer: ```suggestion
   5. When that the `worker` container on your autokit host has started, the logs in the dashboard should read the message `worker setup completed`. If the container is not starting, the most likely cause could be either the hardware not plugged in correctly, or the needed env vars weren't set up correctly.
```
